oio/cli/admin/item_repair.py

```diff
@@ -132,9 +132,27 @@ class ContainerRepair(ContainerCommandMixin, ItemRepairCommand):
     def get_parser(self, prog_name):
         parser = super(ContainerRepair, self).get_parser(prog_name)
         self.patch_parser(parser)
+
+        parser.add_argument(
+            '--no-rebuild-bases', action='store_true',
+            help='Don\'t rebuild the missing, lost bases. '
+                 '(default=%s)' % self.repairer_class.DEFAULT_NO_REBUILD_BASES)
+        parser.add_argument(
+            '--no-sync-bases', action='store_true',
+            help='Don\'t synchronize its bases. '
+                 '(default=%s)' % self.repairer_class.DEFAULT_NO_SYNC_BASES)
+        parser.add_argument(
+            '--no-update-account', action='store_true',
+            help='Don\'t update the counters for the account service. '
+                 '(default=%s)'
+            % self.repairer_class.DEFAULT_NO_UPDATE_ACCOUNT)
         return parser
 
     def _take_action(self, parsed_args):
+        self.conf['no_rebuild_bases'] = parsed_args.no_rebuild_bases
+        self.conf['no_sync_bases'] = parsed_args.no_sync_bases
+        self.conf['no_update_account'] = parsed_args.no_update_account
```

Comment:
Using positive words seems less brain-teasing: `rebuild_bases` instead of `no_rebuild_bases`.
The parser can be changed to use `action="store_false, dest="rebuild_bases"`.

Idem for `sync_bases` and `update_account`.